utils/assetTypes.ts

```diff
@@ -0,0 +1,95 @@
+import { isAddress } from 'web3-utils'
+
+const DOCUMENT_FILE_TYPES = [
```

Comment:
Not sure if you need it, but there is a npm mime or mime-types module which can map mime types to extensions and back. If you have the first 4k of a document, you can use the file-type npm to detect the type of file it is (at least in theory :) )

Reply:
Oh yes that would be very useful, thanks for the hint I'll take look into this packages.